File: C++/1115.cpp
#include <iostream>
#include <string>
#include <limits>
#include <cmath>
#include <iomanip>

using namespace std;

int main()
{
    int x = -1, y = -1;
    cin >> x >> y;

    while ((x != 0) && (y != 0))
    {
        

        if (x > 0 && y > 0)
        {
            cout << "primeiro\n";
        }
        else if (x < 0 && y > 0)
        {
            cout << "segundo\n";
        }
        else if (x < 0 && y < 0)
        {
            cout << "terceiro\n";
        }
        else
        {
            cout << "quarto\n";
        }
        cin >> x >> y;
    }
}

File: JS/URI/1115.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let j = 0;
let x = parseInt(lines[j].split(" ")[0]);
let y = parseInt(lines[j].split(" ")[1]);

while (x != 0 && y != 0) {
    if (x > 0 && y > 0) {
        console.log("primeiro");
    }
    else if (x < 0 && y > 0) {
        console.log("segundo");
    }
    else if (x < 0 && y < 0) {
        console.log("terceiro");
    }
    else {
        console.log("quarto");
    }
    j++;
    x = parseInt(lines[j].split(" ")[0]);
    y = parseInt(lines[j].split(" ")[1]);
}